### ISSUE TITLE
Type de donnée pour les durées

### ISSUE THREAD

**Issue by ccecconi** [DOREMUS-ANR/doremus-ontology#34]
Nous nous interrogeons sur le type de données que nous devons associer au range des propriétés décrivant des durées. 
Il s'agit de data properties mais de quel type : integer, date ou string ?
Ex :  U53 has duration. U78 estimated duration, etc.

**Comment by rtroncy** [comment]
Formellement, le `rdfs:range` de la propriété U53 est `ecrm:E54_Dimension` (bizarrement, la propriété U78 n'a pas de range, est-ce normal ?), ce qui veut dire, une string, l'habitude étant d'écrire une valeur et une unité.

Je propose donc d'utiliser une string et de spécifier l'unité dedans. En général, les durées sont exprimées comment dans vos données ? En secondes toujours ?

**Comment by pasqLisena** [comment]
We currently use `xsd:dayTimeDuration` ([how to use](http://www.datypic.com/sc/xsd/t-xsd_dayTimeDuration.html)).

It is standard and it shoud be compatible with queries (not tested yet).

---
[Query](http://data.doremus.org/sparql?default-graph-uri=&query=select+*+where+%7B+%3Fs+%28mus%3AU53_has_duration%7Cmus%3AU78_estimated_duration%29+%3Fo+%7D+LIMIT+1000&should-sponge=&format=text%2Fhtml&timeout=0&debug=on)

```sparql
SELECT * WHERE {
     ?s (mus:U53_has_duration|mus:U78_estimated_duration) ?o 
} LIMIT 1000
```

Example: http://data.doremus.org/manifestation/ac72da98-b754-3ccf-bb1b-63ed3d086362 (look at the _N3/turtle_ visualisation, there is a bottom button for this). 25 minutes, 43 seconds

```turtle
<http://data.doremus.org/manifestation/ac72da98-b754-3ccf-bb1b-63ed3d086362> mus:U53_has_duration	"PT25M43S"^^xsd:dayTimeDuration .
```

**Comment by rtroncy** [comment]
Right, now I remember we have discussed this @pasqLisena :-) From my point of view, this issue can be closed.

The documentation of the ontology may further describe that our implementation of the property `ecrm:E54_Dimension` for the properties `mus:U53_has_duration` and `mus:U78_estimated_duration` has been to use `xsd:dayTimeDuration`.

We also need to add formally the `rdfs:range` of the property `mus:U78_estimated_duration`. I'm taking this action.
